scaleway/resource_compute_instance_server.go

```diff
@@ -0,0 +1,360 @@
+package scaleway
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+	"github.com/scaleway/scaleway-sdk-go/api/instance/v1"
+	"github.com/scaleway/scaleway-sdk-go/utils"
+)
+
+func resourceScalewayComputeInstanceServer() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceScalewayComputeInstanceServerCreate,
+		Read:   resourceScalewayComputeInstanceServerRead,
+		Update: resourceScalewayComputeInstanceServerUpdate,
+		Delete: resourceScalewayComputeInstanceServerDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+		SchemaVersion: 0,
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Computed:    true,
+				Description: "The name of the server",
+			},
+			"image": {
```

Comment:
You use `security_group_id` but here not `image_id`. Is this done on purpose?

Reply:
Good catch!